src/plugins/janus_videoroom.c

```diff
@@ -13556,7 +13540,7 @@ static void *janus_videoroom_remote_publisher_thread(void *user_data) {
 	g_hash_table_remove_all(publisher->streams_byid);
 	g_hash_table_remove_all(publisher->streams_bymid);
 	janus_mutex_unlock(&publisher->streams_mutex);
-	janus_videoroom_leave_or_unpublish(publisher, TRUE, FALSE);
+	janus_videoroom_leave_or_unpublish(publisher, publisher->session, TRUE, FALSE);
```

Comment:
Is this intended? I don't see the change in leave_or_unpublish's signature

Reply:
The rebase with upstream brought bunch of new code. This call is from new code.

Reply:
Ah ok sorry, I mixed the red and the green